api/data_refinery_api/serializers.py

```diff
@@ -722,6 +729,13 @@ def get_organism_samples(self, obj):
 
         return result
 
+    def get_worker_version(self, obj):
+        processor_jobs = obj.processor_jobs.order_by('-created_at').values_list('worker_version', flat=True)
```

Comment:
I think we should probably filter for `success=True` right?

Reply:
Talked in person, assuming that datasets can only be processed once this will return the correct value for processed datasets. For the datasets that failed to process, it will return the latest version that we used to try to process them, which still might be useful.